### PR TITLE
 Completed express-post-json

### PR DESCRIPTION
**Pull request by Alex-Davis0**
Screenshot 2022-01-25 154606
Screenshot 2022-01-25 154624

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+app.post('/api/grades', (req, res) => {
+  const body = req.body;
+  const grade = {};
+  grade.name = body.name;
+  grade.course = body.course;
+  grade.score = body.score;
+  grade.id = nextId;
+  grades[nextId] = grade;
+  nextId++;
+  res.sendStatus(201);
```
Comment (scbowler) [inline express-post-json/index.js]: Here you want to send back the newly created `grade` to the client.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just missed one thing. Let me know if you have any questions.

**Comment by Alex-Davis0** [comment]
Screenshot 2022-01-25 154624
Screenshot 2022-01-25 160117

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**